static/js/util/util.js

```diff
@@ -386,7 +386,7 @@ export function programCourseInfo(program: Program): Object {
   if (program.courses) {
     totalCourses = program.courses.length;
     passedCourses = program.courses.filter(
-      course => course.runs.length > 0 && course.runs[0].status === STATUS_PASSED
+      course => _.some(course.runs, ["status", STATUS_PASSED])
```

Comment:
This works, but I had to look up the documentation for [`_.some`](https://lodash.com/docs#some) to understand what's going on. Can you add a comment to explain this?
```js
// check if any course run has a `status` property set to STATUS_PASSED
course => _.some(course.runs, ["status", STATUS_PASSED])  
```